### PR TITLE
alert CLI Tool - Flag Usage updated

### PR DESCRIPTION
**Pull request by indrarahul2013**
@vkuznet  alert CLI Tool - Flag Usage updated

### CODE REVIEW

File: src/go/MONIT/alert.go [hunk 1]
```diff
@@ -576,31 +576,31 @@ func main() {
...
+		fmt.Println("\tGet all alerts:")
+		fmt.Println("\t    alert")
+		fmt.Println("\tGet all alerts in JSON format:")
+		fmt.Println("\t    alert -json")
+		fmt.Println("\tGet all alerts with filters (-json flag will output in JSON format if required):")
+		fmt.Println("\t    Available filters:")
+		fmt.Println("\t    service\tEx GGUS,SSB,dbs,etc.")
+		fmt.Println("\t    severity\tEx info,medium,high,urgent,etc.")
+		fmt.Println("\t    tag\t\tEx cmsweb,cms,monitoring,etc.")
+		fmt.Println("\n\t    Get all alerts of specific service/severity/tag. Ex GGUS/high/cms:")
```
Comment (vkuznet) [inline src/go/MONIT/alert.go:588]: I'm not sure if you need additional tag/spaces here. I would suggest that you align all "Get" messages on the same column.

Comment (indrarahul2013) [reply src/go/MONIT/alert.go:588]: Valentin, 
The tabbed "Get" messages are sub-messages. Should I still align them in same column?

Comment (vkuznet) [reply src/go/MONIT/alert.go:588]: All you gets, are just examples, do not complicate things where it is not necessary, it only leads to confusion from user side

Comment (indrarahul2013) [reply src/go/MONIT/alert.go]: Okay. I have made changes.

**Comment by indrarahul2013** [comment]
@vkuznet, now you can merge.

**Pull request merged**